<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HCAH-L-PF19KTHT" failures="1" tests="2" name="Default test" time="0.434" errors="0" timestamp="2021-02-09T18:08:54 IST">
  <testcase classname="CustomerApp_Package.TestScripts_Folder.PatientScripts.CreatePatient" name="createNewPatient" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public java.lang.String CustomerApp_Package.TestScripts_Folder.PatientScripts.CreatePatient.payload() throws java.io.IOException,org.json.simple.parser.ParseException must return either Object[][] or Object[] or Iterator&amp;lt;Object[]&amp;gt; or Iterator&amp;lt;Object&amp;gt;, not class java.lang.String">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String CustomerApp_Package.TestScripts_Folder.PatientScripts.CreatePatient.payload() throws java.io.IOException,org.json.simple.parser.ParseException must return either Object[][] or Object[] or Iterator<Object[]> or Iterator<Object>, not class java.lang.String
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:174)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createNewPatient -->
  <testcase classname="CustomerApp_Package.TestScripts_Folder.PatientScripts.CreatePatient" name="test1" time="0.004"/>
</testsuite> <!-- Default test -->
